extensions/amp-analytics/amp-analytics.md

```diff
@@ -352,6 +352,23 @@ Notice that selector can be used to only specify a single element, not a collect
 The element visibility trigger waits for the signal specified by the `waitFor` property in `visibilitySpec` before tracking element visibility. If `waitFor` is not specified, it waits for element's [`ini-load`](#initial-load-trigger) signal. See `waitFor` docs for more details.
 
 
+##### Error trigger
+
+The user error event (`"on": "user-error"`) is triggered when an error occurs that is attributable to the author of the page or to software that is used in publishing the page. This includes, but not limited to, misconfiguration of an AMP component, misconfigured ads, or failed assertions. User errors are also reported in the developer console.
```

Comment:
to @rudygalfi 
I know we collect user error from both 1p and 3p. Do we want to identify them with different event name later? One possible usage is that publishers can sample 1p error and 3p error at different sample rate?

Reply:
Perhaps not necessary at this stage, but yes, a variable for error type (1P vs 3P) and the possibility of a userErrorSpec that could enable different sampling strategies would be good future considerations.